CONTRIBUTING.md

```diff
@@ -1,20 +1,49 @@
 How to contrubute to SunPy
 --------------------------
 
+Hello, and thanks for being interested in contributing to SunPy!
+
 SunPy is an open project that enourages everyone to contribute in any way possible.
-This document descibes how to contribute code and documentation to the main SunPy 
-repository, if you wish to make a different contribution please contact the 
-[mailing list](https://groups.google.com/forum/#!forum/sunpy).
+This document mainly focuses on code or documentation contributions, however
+there are many other ways to get involved with the SunPy community, a good place
+to start is saying hello on the [mailinglist]() or [IRC]().
+
+
+## Reporting Issues
+
+If you have found a problem when using SunPy the best way to report it and get
+some help is by opening an [issue](https://github.com/sunpy/sunpy/issues) on our
+GitHub issue tracker.
```

Comment:
Add "You will need to sign in to GitHub.com to report an issue.  If you are not already a member of Github.com, you will have to join.  Joining GitHub will also make it easier to report and track issues in the future".

If you do not want to join Github, then another way to report your issue is email the SunPy developers list sunpy-dev@googlegroups.com.